packages/web-client/app/services/layer2-network.ts

```diff
@@ -114,6 +114,9 @@ export default class Layer2Network
       customizationDid,
       options
     );
+
+    taskFor(this.viewSafes).perform(this.walletInfo.firstAddress!);
```

Comment:
Took me a while to understand that this was necessary because a consumer was reading the last successful value of this task, do you think a comment about this would be good?

Reply:
Would it be clearer if we wrapped it in a "refreshSafes()` method?

Reply:
Merging as is, but let's do either a comment or intention-revealing method in a separate PR, whichever you like best.

Reply:
ya, I don’t love the way this works at the moment; the other place this is invoked is a `did-insert` in `balances.hbs`. Both of them pass the `firstAddress` like this. So `refreshSafes` would be a nice encapsulation, I’ll do that, thanks.